brave/brave-core
# fix(iOS): Add 1.73 translation

Resolves https://github.com/brave/brave-browser/issues/42622
Resolves https://github.com/brave/brave-browser/issues/42181



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
Test plan:

1.  Set your device to any language that Brave Support
2. Open Brave Browser App
3. Observe all copies should be localized correctly

Note: in 1.73 we will introduce support for Hebrew, Thai and Vietnamese. Can be verified in other issue https://github.com/brave/brave-browser/issues/42181
